helm/fluent-logshipping-app-chart/templates/fluentbit/service.yaml

```
@@ -6,6 +6,11 @@ metadata:
   labels:
     app: {{ .Values.fluentbit.name }}
   annotations:
+    {{- if .Values.giantswarm_supporting }}
```

```suggestion
    {{- if .Values.giantswarm.supporting }}
```

Why use an underscore here? We should just structure the values correctly.